middle_end/flambda/types/structures/function_declaration_type.rec.ml

```diff
@@ -25,35 +25,41 @@ module Inlinable = struct
     dbg : Debuginfo.t;
     rec_info : Rec_info.t;
     is_tupled : bool;
+    force_inline : bool;
   }
 
-  let print ppf { code_id; dbg; rec_info; is_tupled; } =
+  let print ppf { code_id; dbg; rec_info; is_tupled; force_inline } =
     Format.fprintf ppf
       "@[<hov 1>(Inlinable@ \
         @[<hov 1>(code_id@ %a)@]@ \
         @[<hov 1>(dbg@ %a)@] \
         @[<hov 1>(rec_info@ %a)@]\
         @[<hov 1><is_tupled@ %b)@]\
+       @[<hov 1><force_inline@ %b)@]\
         )@]"
```

Comment:
This format string isn't right (there were some existing problems too).  Here is a corrected version:
```
diff --git a/middle_end/flambda/types/structures/function_declaration_type.rec.ml b/middle_end/flambda/types/structures/function_declaration_type.rec.ml
index 205eb273df..3c92bd5d80 100644
--- a/middle_end/flambda/types/structures/function_declaration_type.rec.ml
+++ b/middle_end/flambda/types/structures/function_declaration_type.rec.ml
@@ -32,10 +32,10 @@ module Inlinable = struct
     Format.fprintf ppf
       "@[<hov 1>(Inlinable@ \
         @[<hov 1>(code_id@ %a)@]@ \
-        @[<hov 1>(dbg@ %a)@] \
-        @[<hov 1>(rec_info@ %a)@]\
-        @[<hov 1><is_tupled@ %b)@]\
-       @[<hov 1><force_inline@ %b)@]\
+        @[<hov 1>(dbg@ %a)@]@ \
+        @[<hov 1>(rec_info@ %a)@]@ \
+        @[<hov 1>(is_tupled@ %b)@]@ \
+        @[<hov 1>(force_inline@ %b)@]\
         )@]"
       Code_id.print code_id
       Debuginfo.print_compact dbg
```